doc/api/fs.md

```diff
@@ -2066,16 +2066,22 @@ changes:
 -->
 
 * `path` {string|Buffer|URL}
-* `mode` {integer} Not supported on Windows. **Default:** `0o777`.
+* `options` {Object|string}
```

Comment:
{Object|integer}

applies also to `fs.mkdirSync` and `fsPromises.mkdir`

Reply:
It's not clear that the `integer` value is still supported. It should.